core/src/main/scala/filodb.core/memstore/TimeSeriesPartition.scala

```diff
@@ -508,7 +508,19 @@ TimeSeriesPartition(partID, schema, partitionKey, shard, bufferPool, shardStats,
   override def switchBuffers(blockHolder: BlockMemFactory, encode: Boolean = false): Boolean = {
     _log.info(s"SwitchBuffers dataset=$ref schema=${schema.name} shard=$shard partId=$partID $stringPartition - " +
                s"encode=$encode for currentChunk ${currentInfo.debugString}")
-    super.switchBuffers(blockHolder, encode)
+    val ret = super.switchBuffers(blockHolder, encode)
```

Comment:
Can we comment out the chunkmapAcquireShared and related methods ? Not needed anymore and makes logging too verbose

Reply:
done